lib/api/v1/controllers/keywords_test.go

```diff
@@ -19,8 +21,12 @@ var _ = Describe("KeywordsController", func() {
 
 		Context("Given VALID payload", func() {
 			It("returns the status OK", func() {
-				fabricators.FabricateUser("test@gmail.com", "123456")
-				ctx, resp := test.MakeMultipartRequestRequest("/keywords/upload", "keywords/valid.csv", "text/csv")
+				user := fabricators.FabricateUser("test@gmail.com", "123456")
+				headers := http.Header{}
+				headers.Set("Authorization", "Bearer "+helpers.GenerateToken(user.Base.ID.String()))
```

Comment:
I think we can do this
```suggestion
				headers := http.Header{
					"Authorization": {"Bearer "+helpers.GenerateToken(user.Base.ID.String())} }
```

Reply:
If you agree with this, please update for other test cases